generated/artifacts/openapi-yaml/openapi/openapi.yaml

```diff
@@ -1982,6 +1825,8 @@ components:
       - report_name
       type: object
     id_photo:
+      allOf:
+      - $ref: '#/components/schemas/read_only_2'
```

Comment:
Better to add prefixing that with folder name to avoid ambiguity

Reply:
Took the opportunity to rename as follows:
- `common` -> `shared`
- `write_only` -> `request`
- `read_only` -> `response`